definitions/npm/enzyme_v3.x.x/flow_v0.53.x-/enzyme_v3.x.x.js

```diff
@@ -49,6 +49,7 @@ declare module "enzyme" {
     prop(key: string): any,
     key(): string,
     simulate(event: string, ...args: Array<any>): this,
+    simulateError(error: any): this,
```

Comment:
Isn't it a bit sad to use `any` here? 
The method calls `componentDidCatch` under the hood.

Reply:
In [docs](https://github.com/airbnb/enzyme/blob/944f9e098bb1978c750525416a18a58c80fbe126/docs/api/ReactWrapper/simulateError.md#arguments), the type of error is `any`.

I had traced the [code](
https://github.com/airbnb/enzyme/blob/02d63378a36a74c4e76f74a40404b218b9905e66/packages/enzyme-adapter-utils/src/Utils.js#L281), it call `componentDidCatch` directly. In `flow`, the type of error is `Error` in [componentDidCatch](https://github.com/facebook/flow/blob/master/lib/react.js#L77-L82).

Maybe `Error` is more better?

Reply:
I would say so yes.

Reply:
OK